app/assets/javascripts/views/task_form.js

```diff
@@ -40,7 +40,7 @@ Fulcrum.TaskForm = Fulcrum.FormView.extend({
       },
 
       error: function(model, response) {
-        if(response.responseText === undefined) return;
+        if(!response.responseText || response.responseText == '') return;
```

Comment:
To grant the value of the string is really empty, use a trim method.
`!response.responseText.trim()`